Annex60/Fluid/Movers/BaseClasses/PartialFlowMachine.mo

```diff
@@ -25,7 +22,7 @@ partial model PartialFlowMachine
     "Set to false to avoid any power (=heat and flow work) being added to medium (may give simpler equations)";
 
   parameter Modelica.SIunits.Time tau=1
-    "Time constant of fluid volume for nominal flow, used if dynamicBalance=true"
```

Comment:
@rubenbaetens This comment is wrong. It should be "... used if energyDynamics <> Modelica.Fluid.Types.Dynamics.SteadyState" as there are three different energy dynamics configurations, and users typically set energyDynamics as this automatically assigns the same value for massDynamics.

Also, you need to fix the annotation `enable=dynamicBalance`